services/requester/requester.go

```diff
@@ -76,6 +76,9 @@ type Requester interface {
 	// GetCode returns the code stored at the given address in
 	// the state for the given block number.
 	GetCode(ctx context.Context, address common.Address, height uint64) ([]byte, error)
+
+	// GetCode returns the latest block height of the Flow network.
+	GetLatestBlockHeight(ctx context.Context) (uint64, error)
```

Comment:
Name it more explicit like: `GetLatestCadenceHeight` 
Also update the comment to consistently say Cadence Height since this is now used all over the codebase to mean the Flow FVM Block height.

Reply:
Fair point :+1: Updated in https://github.com/onflow/flow-evm-gateway/pull/180/commits/0060f038b0fa62522b12b678c5dafc044b746699